pkg/agent/cniserver/server.go

```diff
@@ -455,9 +445,11 @@ func (s *CNIServer) CmdDel(_ context.Context, request *cnipb.CniCmdRequest) (
 	// Remove host interface and OVS configuration
 	podName := string(cniConfig.K8S_POD_NAME)
 	podNamespace := string(cniConfig.K8S_POD_NAMESPACE)
-	if err := s.podConfigurator.removeInterfaces(podName, podNamespace, cniConfig.ContainerId); err != nil {
-		klog.Errorf("Failed to remove interfaces for container %s: %v", cniConfig.ContainerId, err)
-		return s.configInterfaceFailureResponse(err), nil
+	if ipam.IsInfraContainer(cniConfig.Netns) {
```

Comment:
Is there such case? I think not according to the spec?

Reply:
The check is used when call CmdDel as failure callback.

I cam this after I change the behavior of callback as above comments.

Done.